### DISCUSSION TITLE
Tips for New Students: Making the Most of Global Campus

### DISCUSSION THREAD

**Discussion by A-Abdeen** [community/community#128574] [GitHub Education]
Hi everyone,

I have just joined the Global Campus. I was recommended to start with the Deep Dive learning path (GitHub Flow + Codespaces)

As a student, I haven't specialized yet, I am mostly doing Full Stack Development with Go & JS for now, and I will be learning Architecture and Rust in the near future.

My questions are:

- How did you make the most out of the available resources & skills?
- Is there a certain path/skill you recommend?
- What would you consider a "hidden gem" among the offers in the Student Development Pack?

Feel free to share any additional tips / experiences!

**Comment by ebndev** [comment]
Hi @A-Abdeen, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our GItHub Education category, as this topic might be more relevant there 😄
